src/main/java/fruitshop/processingdata/MainShop.java

```diff
@@ -0,0 +1,41 @@
+package fruitshop.processingdata;
+
+import fruitshop.database.TransactionService;
+import fruitshop.dataservice.DataServiceImp;
+import fruitshop.dataservice.SupplierReport;
+import fruitshop.dataservice.WorkWithFile;
+import fruitshop.dataservice.WorkWithReport;
+import fruitshop.model.FruitTransaction;
+import fruitshop.model.Operation;
+import fruitshop.model.SaveData;
+import fruitshop.strategy.BalanceActivity;
+import fruitshop.strategy.PurchaseActivity;
+import fruitshop.strategy.SupplyActivity;
+import fruitshop.strategy.TypeOfActivity;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class MainShop {
+    private static final String readFromFileName = "src/main/resources/dayFile.csv";
+    private static final String writeToFileName = "src/main/resources/fileToreport.csv";
+
+    public static void main(String[] args) {
+        Map<Operation, TypeOfActivity> operationHandlerMap = new HashMap<>();

```

Comment:
I think you should to extract map initialization and operationHandlerMap.put() from main method to improve readability. For example in static block or create static method

Reply:
I am agree